lib/src/mock_collection_reference.dart

```diff
@@ -26,7 +27,8 @@ class MockCollectionReference extends MockQuery implements CollectionReference {
   final String _path;
 
   // ignore: unused_field
-  final CollectionReferencePlatform _delegate = null;
+  final CollectionReferencePlatform _delegate =
```

Comment:
Cannot be null anymore since the field in Platform Interface is non-nullable, so I made a Mock.